K2Bridge/Visitors/HistogramVisitor.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+namespace K2Bridge.Visitors
+{
+    using System;
+    using Utils;
+    using K2Bridge.Models.Request.Aggregations;
+
+    /// <content>
+    /// A visitor for the <see cref="HistogramAggregation"/> element.
+    /// </content>
+    internal partial class ElasticSearchDSLVisitor : IVisitor
+    {
+        /// <inheritdoc/>
+        public void Visit(HistogramAggregation histogramAggregation)
+        {
+            Ensure.IsNotNull(histogramAggregation, nameof(histogramAggregation));
+            EnsureClause.StringIsNotNullOrEmpty(histogramAggregation.Metric, nameof(histogramAggregation.Metric));
+            EnsureClause.StringIsNotNullOrEmpty(histogramAggregation.Field, nameof(histogramAggregation.Field));
+
+            var histogramKey = EncodeKustoField(histogramAggregation.Key + "%" + histogramAggregation.Keyed);
+
+            histogramAggregation.KustoQL = $"{KustoTableNames.Data} | {KustoQLOperators.Summarize} {histogramAggregation.SubAggregationsKustoQL}" +
+            $"{histogramAggregation.Metric} by {histogramKey} = ";
+
+            var interval = Convert.ToInt32(histogramAggregation.Interval);
+            var field = EncodeKustoField(histogramAggregation.Field, true);
+
+            histogramAggregation.KustoQL += $"bin({field}, {interval})";
+
+            // todatetime is redundent but we'll keep it for now
```

Comment:
Can you remove this comment? it's obsolete any (there's no todatetime).

Reply:
Done